curs_main.c

```diff
@@ -50,6 +50,7 @@
 #include "protos.h"
 #include "sort.h"
 #include "thread.h"
+#include <term.h>
```

Comment:
I guess this is for `tigetstr`, etc.
`<term.h>` is part of the curses install but doesn't exist in a slang build.

Putting this around it might do, for now:
```
#if defined(HAVE_NCURSESW_NCURSES_H) || defined(HAVE_NCURSES_H) || defined(HAVE_NCURSES_NCURSES_H)
#endif
```